EnglishTranslation-master/QUEST_LV_0100.tsv

```diff
@@ -12321,252 +12321,252 @@ QUEST_LV_0100_20151224_012321	But as you know.. {nl}There are barely enough sold
-QUEST_LV_0100_20151224_012325	Forget the monsters going wild because of the stench, but the fact that they're gathering near Orsha is definitely not natural.{nl}I don't know why they are acting like that. Do you think there may be demons somewhere?
-QUEST_LV_0100_20151224_012326	At least there won't be any more monsters in my area of responsibility for a while.{nl}Thank you for helping out.
-QUEST_LV_0100_20151224_012327	Hey there, how's it going?{nl}By the looks of things, you've also recieved some requests from that person over there..{nl}
-QUEST_LV_0100_20151224_012328	Do you want something really useful?{nl}It's called Monster Stimulant, and monsters sprayed with the stuff will attack surrounding monsters indiscriminately.{nl}
-QUEST_LV_0100_20151224_012329	Why don't you try it out?{nl}It will let you pass on some of the more irksome work.
-QUEST_LV_0100_20151224_012330	Medzio Diena is a terrible memory for everyone.{nl}My hometown had been destroyed that fateful day, and I'm living off these odd jobs in Orsha.
-QUEST_LV_0100_20151224_012331	What do you think?{nl}Isn't it really effective?{nl}
-QUEST_LV_0100_20151224_012332	I dreamt of becoming an alchemist before all of this happened.{nl}This is actually a failure since I only learned over the shoulder's of others, but I guess it's all good if you can find a use for it.
-QUEST_LV_0100_20151224_012333	By the goddesses, I've never seen anything like it.{nl}I went up Bonan Forest Road because it felt as if the monsters were pouring out from there.{nl}
+QUEST_LV_0100_20151224_012325	Forget the monsters going wild because of the stench, {nl}but the fact that they're gathering near Orsha is definitely not natural.{nl}I don't know why they are acting like that. {nl}Do you think there may be demons somewhere?
+QUEST_LV_0100_20151224_012326	At least I won't be responsible for any more monsters in my area for a while.{nl}Thank you for helping out.
+QUEST_LV_0100_20151224_012327	Hey there, how's it going? {nl}It appears that you've also received requests from that person over there...{nl}
+QUEST_LV_0100_20151224_012328	Do you want something really useful? It's a Monster Stimulant.{nl}Monsters sprayed with it will become hostile and attack other nearby monsters.{nl}
+QUEST_LV_0100_20151224_012329	Why don't give it a try?{nl}It will let you pass on some of the more irksome work.
+QUEST_LV_0100_20151224_012330	Medzio Diena is a terrible memory for everyone.{nl}My hometown had been destroyed on that fateful day,{nl}and I'm living off these odd jobs in Orsha.
+QUEST_LV_0100_20151224_012331	What do you think? Isn't it really effective?{nl}
+QUEST_LV_0100_20151224_012332	I dreamt of becoming an alchemist before all of this happened.{nl}This is actually a failure since I only learned over the shoulder's of others, {nl}but I guess it's all good if you can find a use for it.
```

Comment:
"This is actually a failure since I only learned over the shoulder's of others, {nl}but I guess it's all good if you can find a use for it."
`I only learned it by watching over the shoulders of others so this is actually failure, {nl}but I guess it's all good if you can find a use for it.`